cms/static/js/factories/manage_users.js

```diff
@@ -9,7 +9,7 @@ function(_, gettext, ManageUsersAndRoles) {
             var local_messages = _.extend({}, messages);
             local_messages.alreadyMember.title = gettext('Already a course team member');
             local_messages.deleteUser.messageTpl = gettext(
-                'Are you sure you want to delete {email} from the course team for “{container}”?'
+                'Are you sure you want to delete <%= email %> from the course team for “<%= container %>”?'
```

Comment:
@cahrens @dan-f I prefer the interpolation syntax of ``{email}`` and I'll try to explain why below.  Given that, I do think that the name ``messageTpl`` is misleading because I would expect this to be a template, which is typically an Underscore template.  I'm not sure how painful it would be to rename this to ``interpolateMessage``, or something of that sort.  I'm not saying you must refactor that name here, but I'm also trying to establish names and conventions for our documentation, so it would be nice to determine what we would do if this were new code.

The reason I prefer the ``{email}`` syntax is because I think it is easier to get tripped up by ``<%-`` vs ``<%=`` in general.  The rule I am trying to establish is to use ``<%-`` almost everywhere, which will perform escaping.  Since we must have an exception, the current exception would be that ``<%= safe_interpolate('...') %>`` would also be an acceptable.

For individual variables, I'd really like to avoid having to state that ``<%- email ->`` makes sense in most cases, but ``<%= email ->`` makes sense in other cases.  It's my opinion that it will be too difficult to remember the right one to use, and you'd either be left with unsafe variables in some cases or double-escaped in others.